lib/dpul_collections_web/live/search_live.ex

```diff
@@ -171,7 +171,7 @@ defmodule DpulCollectionsWeb.SearchLive do
       navigate={self_route(@search_state, %{facet: %{@field => nil}})}
       class="filter mb-2 focus:border-3 focus:visible:border-rust focus:border-rust py-2 px-4 shadow-md no-underline rounded-lg bg-primary border-dark-blue text-white font-sans font-semibold text-sm btn-primary hover:text-white hover:bg-accent focus:outline-none active:shadow-none"
     >
-      {@label}
+      {Gettext.gettext(DpulCollectionsWeb.Gettext, @label)}
```

Comment:
This is okay as long as they're spelled out somewhere else; maybe you could add a comment that says where in the code they are enumerated?